README.md

```diff
@@ -44,3 +44,49 @@ Run gradle build command either using local distribution or wrapper
 
 Generated war will be available under `BackendWebApp/builds/libs` folder. 
 
+## Getting Started
```

Comment:
in order to keep this fail short, we can add getting started in indepent doc and create an index in this. What do you think?

Reply:
Fixed